README.md

```diff
@@ -17,9 +17,58 @@ Here's one way you can get all the right versions installed and setup:
 2. `nvm use`
    -  run this command in the project root to install compatible versions of `node` and `npm`
 3. `npm install -g pnpm@8`
+4. `npm install -g yarn`
 
 ### Setup
 
+#### Setup Verdaccio Auth Token
+
+We use [Verdaccio](https://verdaccio.org/) as our private npm registry. To install packages from Verdaccio, you will need to setup an authentication token.
+
+⚠️ **_You will need to be IP whitelisted to be able to access our Verdaccio registry._**
+
+<br/>
+
+**New Users** <br/>
+If you've never created a Verdaccio account, you can follow these steps:
+
+1. Copy `.env.local.example` to `.env.local`
+   ```
+   cp .env.local.example .env.local
+   ```
+2. Run
+   ```
+   pnpm adduser --registry=https://verdaccio.ubreakit.com
+   ```
+   -  _You will then be prompted to enter some information_
+3. Enter a `username`
+4. Enter a `password` - _make it rememberable and strong_
+5. Enter an `email address`
+
+**Login** <br/>
+If you've already created a Verdaccio account, and need to grab the token again, you can follow these steps:
+
+1. Run
+   ```
+   pnpm login --registry=https://verdaccio.ubreakit.com
+   ```
+2. Enter your `username`
+3. Enter your `password`
+
+<br/>
+
+**Setting the Env Token** <br/>
+Once you've signed up, or logged in, the Verdaccio auth token will be saved to your `~/.npmrc`. You can grab the token from there and save it to `.env.local` as `VERDACCIO_AUTH_TOKEN`.
+
+<br/>
+
+⚠️ **_If you don't want to use the Verdaccio registry outside of this repo, you can run the following:_**
+
+-  ```
+   pnpm logout --registry=https://verdaccio.ubreakit.com
+   ```
+   -  This will remove the Verdaccio auth token from your `~/.npmrc`, and so long as you've exported the token to `.env.local`, you will still be able to install packages from Verdaccio for this repo.
```

Comment:
> Is all of this really necessary? It be nice to keep the setup process as simple as we possibly can. Can we make the token easily available somewhere without requiring setting up accounts?

We could reduce the security by using a single token for every Dev user if that's what would be preferred.

> For context, are we using a private registry because of font awesome private registry rate limit, or are there other reasons?

That was the main reason but this also helps to create an additional caching layer for CI.

Reply:
> We could reduce the security by using a single token for every Dev user if that's what would be preferred.

I'd vote for that 👍

Reply:
Gotcha, I'll update the README accordingly. I'll still leave these instructions under a `toggle` section in case other folks want to create their own token.

Reply:
Should be addressed with [c77de29](https://github.com/iFixit/react-commerce/pull/1822/commits/c77de29400c249e0f423dd7474f869f9a02b3eb2)